pom.xml

```diff
@@ -6,13 +6,14 @@
 
     <groupId>com.smartling.marketo</groupId>
     <artifactId>marketo-sdk</artifactId>
-    <version>3.1.24</version>
+    <version>3.1.25-SNAPSHOT</version>
```

Comment:
Why did you decide to use snapshot version?

Reply:
This version for the branch. I'll make release version after merge